cmd/builder/test/test.sh

```diff
@@ -58,8 +56,7 @@ test_build_config() {
     retries=0
     while true
     do
-        kill -0 "${pid}" >/dev/null 2>&1
-        if [ $? != 0 ]; then
+        if ! kill -0 "${pid}" >/dev/null 2>&1; then
```

Comment:
Should this be `if kill` since we want to enter this condition only when the return code is not 0

Reply:
I think it shouldn't be `if kill`
From https://www.shellcheck.net/wiki/SC2181:
"To check that a command returns failure, use `if ! mycommand; then ...` ."